crates/e2e_test/testdata/docker-compose.yaml

```diff
@@ -0,0 +1,59 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+version: '3.8'
+
+services:
+  rest:
+    image: tabulario/iceberg-rest:0.10.0
+    environment:
+      - AWS_ACCESS_KEY_ID=admin
+      - AWS_SECRET_ACCESS_KEY=password
+      - AWS_REGION=us-east-1
+      - CATALOG_CATOLOG__IMPL=org.apache.iceberg.jdbc.JdbcCatalog
+      - CATALOG_URI=jdbc:sqlite:file:/tmp/iceberg_rest_mode=memory
+      - CATALOG_WAREHOUSE=s3://icebergdata/demo
+      - CATALOG_IO__IMPL=org.apache.iceberg.aws.s3.S3FileIO
+      - CATALOG_S3_ENDPOINT=http://minio:9000
+    depends_on:
+      - minio
+    links:
+      - minio:icebergdata.minio
+    expose:
+      - 8181
+
+  minio:
```

Comment:
It doesn't boot on my end:

```
➜  iceberg-rust git:(tx_append) docker logs -f d7a12d1f9d30
Formatting 1st pool, 1 set(s), 1 drives per set.
WARNING: Host local has more than 0 drives of set. A host failure will result in data becoming unavailable.
MinIO Object Storage Server
Copyright: 2015-2024 MinIO, Inc.
License: GNU AGPLv3 <https://www.gnu.org/licenses/agpl-3.0.html>
Version: RELEASE.2024-03-07T00-43-48Z (go1.21.8 linux/arm64)

Status:         1 Online, 0 Offline. 
S3-API: http://172.20.0.2:9000  http://127.0.0.1:9000     
Console: http://172.20.0.2:9001 http://127.0.0.1:9001   

Documentation: https://min.io/docs/minio/linux/index.html
Warning: The standard parity is set to 0. This can lead to data loss.

API: ListObjectsV2(bucket=icebergdata)
Time: 12:11:34 UTC 04/25/2024
DeploymentID: 0d2c88aa-2393-4c17-a28c-560e6cfe4b9b
RequestID: 17C984C03C8E1AFA
RemoteHost: 172.20.0.3
Host: minio:9000
UserAgent: MinIO (linux; arm64) minio-go/v7.0.67 mc/RELEASE.2024-03-07T00-31-49Z
Error: volume not found (cmd.StorageErr)
       7: internal/logger/logonce.go:118:logger.(*logOnceType).logOnceIf()
       6: internal/logger/logonce.go:149:logger.LogOnceIf()
       5: cmd/erasure-server-pool.go:1516:cmd.(*erasureServerPools).ListObjects()
       4: cmd/erasure-server-pool.go:1275:cmd.(*erasureServerPools).ListObjectsV2()
       3: cmd/bucket-listobjects-handlers.go:210:cmd.objectAPIHandlers.listObjectsV2Handler()
       2: cmd/bucket-listobjects-handlers.go:156:cmd.objectAPIHandlers.ListObjectsV2Handler()
       1: net/http/server.go:2136:http.HandlerFunc.ServeHTTP()

 You are running an older version of MinIO released 1 month before the latest release 
 Update: Run `mc admin update ALIAS` 
```

Reply:
how about change `minio/minio:RELEASE.2024-03-07T00-43-48Z`, `minio/mc:RELEASE.2024-03-07T00-31-49Z` to `minio/minio:latest`, `minio/mc:latest`.

Reply:
I'm not sure if the error is relevant. I'm able to run it locally now.

> how about change `minio/minio:RELEASE.2024-03-07T00-43-48Z`, `minio/mc:RELEASE.2024-03-07T00-31-49Z` to `minio/minio:latest`, `minio/mc:latest`.

+1 on that. I haven't encountered any issues with minio updates